UnityProject/Assets/Scripts/Objects/Disposals/DisposalVirtualContainer.cs

```diff
@@ -169,8 +169,11 @@ private void ThrowContainedItems(Vector3 throwVector)
 		{
 			foreach (ObjectBehaviour item in containedItems)
 			{
+				if(item == null) continue;
 				ThrowItem(item, throwVector);
 			}
+
+			containedItems.Clear();
```

Comment:
Should be cleared in `EjectContentsAndThrow()`

Reply:
ah yeah missed that